API/app/Models/Form.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
+
+use App\Traits\OpisUtilities;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Eloquent\Model;
+
+class Form extends Model
+{
+    use OpisUtilities; 
+
+    protected $table = 'schede'; 
+
+    public $timestamps = false; 
+
+    /**
+     * @return array 
+     */
+    public static function getAllByAcademicYear (string $academicYear) : array
```

Comment:
Non ricordo se in PHP 7.4 puoi definire il tipo di array tipo cosi':
```PHP
public function test(): schede[]
```
ma sarebbe ottimo (in typescript si puo' fare, in PHP 7.2 mi sembra di no ma magari si puo' fare nelle versioni successive).

Reply:
Per adesso lascerei array, mi informerò sulla sintassi di ritorno. Non che PHP aiuti particolarmente 😂😂